loader/loader_common.h

```diff
@@ -230,6 +232,102 @@ struct loader_instance_dispatch_table {
 // Unique magic number identifier for the loader.
 #define LOADER_MAGIC_NUMBER 0x10ADED010110ADEDUL
 
+struct loader_log_settings {
+    uint32_t enabled_log_flags;  // What log messages to pay attention to
+
+    // Where messages should go to.  More than one of these may be enabled.
+    bool log_errors_to_stdout;     // Log all error output to stdout
+    bool log_errors_to_stderr;     // Log all error output to stderr (default)
+    bool log_nonerrors_to_stdout;  // Log all non-error output to stdout (default)
```

Comment:
It seems to me that the loader handle fine grain of message type so what about using enum flags (that probably already exist for these?)

Reply:
It just complicates the existing logic since we currently check against one flag, now we would have to check flags for each output option.  @charles-lunarg , what's your opinion?

Reply:
Yes we would check for each flags but I can trivially express the flags in vkconfig while this approach is quite inconsistent from the user perspective.